### PR TITLE
doc changes for readability [ci skip]

### PR DESCRIPTION
**Pull request by gouravtiwari**

### CODE REVIEW

File: activejob/lib/active_job/queue_adapter.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def queue_adapter
       end
 
       # Specify the backend queue provider. The default queue adapter
-      # is the :inline queue. See QueueAdapters for more
+      # is the +:inline queue+. See QueueAdapters for more
```
Comment (rafaelfranca) [inline activejob/lib/active_job/queue_adapter.rb]: Should not be just `:inline` inside the fixed font tag?

Comment (gouravtiwari) [inline activejob/lib/active_job/queue_adapter.rb]: oops sorry for typo, fixing it right-away

**Comment by rafaelfranca** [comment]
Could you squash your commits?

**Comment by gouravtiwari** [comment]
@rafaelfranca I squashed my commits.

**Pull request merged**